README.md

```diff
@@ -9,3 +9,4 @@ Repository ini digunakan sebagai sayembara pencarian volunter untuk pembuatan ap
 Berikut daftar Volunter yang diterima:
 
 * Oon Arfiandwi, [oo.or.id](https://oo.or.id).
+* Taufik Romadhon, [Linkedin](https://www.linkedin.com/in/).
```

Comment:
Oke, sudah sesuai formatnya. Terima kasih telah mendaftar! Mohon ditunggu ya, info lanjutan dari Panitia.

Thank you!

Reply:
Sama-sama!